src/components/Footer.tsx

```diff
@@ -0,0 +1,88 @@
+import Divider from '@mui/material/Divider'
+import { FaFacebookF, FaInstagram, FaTiktok, FaYoutube } from 'react-icons/fa'
+import { FaXTwitter } from 'react-icons/fa6'
+import FooterLink from './FooterLink'
+import { createMedia } from '@artsy/fresnel'
+
+const { MediaContextProvider, Media } = createMedia({
+	breakpoints: {
+		sm: 0,
+		md: 768,
+	},
+})
```

Comment:
It's not the best idea to create a media object per each component. You probably should declare it in the theme